src/Benchmarks/Benchmarks.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net451</TargetFrameworks>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
> You guys are crazy, this makes no sense to me

@davidfowl  can you elaborate what doesn't make sense? These kinds of jabs make it hard to move the conversation forward.